### PR TITLE
Responsive Linode Entity Detail ft. legacy plans

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
~Move the status and ActionMenu to the following row for the Linode Landing page in the detailed view~

~I threw in a bunch of logic to better handle the footer for the following scenarios:~
- ~Standard plans with tags~
- ~Standard plans with no tags~
- ~Legacy plans with tags~
- ~Legacy plans with no tags~

**Update:** To simplify things, the tags will wrap to the following row at 1400px for all instances

## Note to Reviewers
~I added some mock Linodes to show the difference, will remove them before I merge this PR. Also, apologies for the magic numbers and logic, open to recommendations on how to better document them 😅 ~

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 1]
```diff
@@ -838,21 +843,32 @@ export const Footer: React.FC<FooterProps> = React.memo(props => {
     [linodeTags, linodeId, updateLinode, enqueueSnackbar]
   );
 
+  const isLegacy = linodePlan && linodePlan.includes('(pending upgrade)');
+
+  const hasTags = linodeTags.length === 0 ? false : true;
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:848]: ternary isn't necessary here (`hasTags = linodeTags.length > 0`)

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 2]
```diff
@@ -838,21 +843,32 @@ export const Footer: React.FC<FooterProps> = React.memo(props => {
     [linodeTags, linodeId, updateLinode, enqueueSnackbar]
   );
 
+  const isLegacy = linodePlan && linodePlan.includes('(pending upgrade)');
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:846]: I forget how this works, but isn't there a more authoritative way to determine this? `linodePlan.successor` or something

**Review by Jskobos — COMMENTED** [review]
The row wrapping looks good. I left some code comments.

I'm...not quite sold on the footer customization. Is this mostly because shadow plans have long labels that break the layout? It's a lot of hard coding just to account for this (rare) case, and then what happens if bare metal plans (for example) have long labels down the road? We could maybe hide the time from date created to add some room in the footer, or reconsider what we're putting in there, or let flex do the wrapping

**Comment by tiffwong** [comment]
@Jskobos good point, I'll remove the legacy styling in the footer for now and just leave it as is for the meantime

**Review by johnwcallahan — APPROVED** [review]
Looks great!

**Review by Jskobos — APPROVED** [review]

**Pull request merged**